src/java/com/google/cloud/bigquery/dwhassessment/extractiontool/db/AvroHelper.java

```diff
@@ -168,6 +173,33 @@ private static void convertColumnTypeToAvroType(
     }
   }
 
+  public static Timestamp getUnadjustedTimestamp(ResultSet row, String columnName)
+      throws SQLException {
+    Calendar cal = Calendar.getInstance();
+    Timestamp timestamp = row.getTimestamp(columnName, cal);
+    return unadjustTimestamp(timestamp, cal);
+  }
+
+  public static Timestamp getUnadjustedTimestamp(ResultSet row, int columnIndex)
+      throws SQLException {
+    Calendar cal = Calendar.getInstance();
```

Comment:
It seems this line can be moved under the `unadjustTimestamp`

Reply:
Actually no, row.getTimestamp(columnIndex, cal) and row.getTimestamp(columnName, cal) actually modify the cal by setting its time zone (surprise, surprise, another anti-pattern), and this time zone information is needed in the unadjustTimestamp()

Reply:
My bad, I didn't notice for some reason that it is also passed to `row.getTimestamp`